src/modify.js

```diff
@@ -60,3 +60,37 @@ exports.modifyService = function(env, serviceName, url, remove) {
     })
   })
 }
+
+exports.modifyMultipleServices = function(env, newImports) {
+  return new Promise((resolve, reject) => {
+    lock.writeLock(releaseLock => {
+      const resultPromise = ioOperations.readManifest(env)
+        .then(data => {
+          let json
+          try {
+            json = data ? JSON.parse(data) : getEmptyManifest()
+          } catch (err) {
+            releaseLock()
+            reject('Could not read import map -- ' + err)
+            return
+          }
+
+          const imports = getMapFromManifest(json)
+          Object.assign(imports, newImports)
+
+          const newImportMapString = JSON.stringify(json, null, 2)
```

Comment:
shouldn't this be `imports` instead of `json`? json is the old one, and imports is the one that was merged in with newImports right?

Reply:
No, returning json is correct. `getMapFromManifest(json)` is just returns the inner `imports` object inside of an import map object. So by modifying`imports`, we are actually modifying an inner property of `json`. And then we return the modified json.

I copied this code from the modifyService function but honestly should have just rewritten it to avoid confusing things like this haha

#mutation

Reply:
ah I see. yeah that's confusing, but makes sense once you explain it.